### ISSUE TITLE
Detect in-progress rebases during repository normalization

### ISSUE THREAD

**Issue by raxod502** [raxod502/straight.el#271]
Within `M-x straight-normalize-package`, we should check if there is a rebase in progress, and handle appropriately (much like we already do for checking if there is a merge in progress). This issue replaces a long-standing FIXME notice in the code for `straight.el`:

https://github.com/raxod502/straight.el/blob/eb2da3dcd590e258578203a112b894fa98e98944/straight.el#L1369-L1388

**Comment by raxod502** [comment]
See [code in Magit](https://github.com/magit/magit/blob/5f29cdb527c43046a843d9d1e760e823b39906f6/lisp/magit-sequence.el#L669-L672):

```elisp
(defun magit-rebase-in-progress-p ()
  "Return t if a rebase is in progress."
  (or (file-exists-p (magit-git-dir "rebase-merge"))
      (file-exists-p (magit-git-dir "rebase-apply/onto"))))
```

**Comment by progfolio** [comment]
Fixed by #711
